src/Xamarin.Android.Build.Tasks/Tasks/CreateDynamicFeatureManifest.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Xml.Linq;
+using Microsoft.Build.Utilities;
+using Microsoft.Build.Framework;
+
+using Xamarin.Android.Tools;
+using Microsoft.Android.Build.Tasks;
+
+namespace Xamarin.Android.Tasks
+{
+	public class CreateDynamicFeatureManifest : AndroidTask
+	{
+		readonly XNamespace androidNS = "http://schemas.android.com/apk/res/android";
```

Comment:
At some point I wonder if we should make these `static` fields 'somewhere"; we *also* have this in `ConvertCustomView.cs`, `IntentFilterAtttribute.Partial.cs`, `GetJavaPlatformJar.cs`, …